pom.xml

```diff
@@ -15,15 +15,15 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <java.version>21</java.version>
-        <spring.boot.version>3.4.0</spring.boot.version>
+        <spring.boot.version>3.3.5</spring.boot.version>
```

Comment:
Is this regression a mistake? Latest atm is 3.4.1

Reply:
Resolved in latest commit